controllers/instancechartrepository_controller.go

```diff
@@ -111,12 +112,6 @@ func (r *InstanceChartRepositoryReconciler) SetupWithManager(mgr ctrl.Manager) e
 		return err
 	}
 
-	// Watch for changes to the secondary resource corev1.Secrets
```

Comment:
Didn't we just add this?

Reply:
Yup, and we immediately dropped it - see [this conversation](https://github.com/mittwald/harbor-operator/pull/52#discussion_r533324388).
Explicitly watching secrets as secondary resource isn't necessary in this controller, which makes this comment obsolete.